### PR TITLE
5605 remove deprecated

### LINKED ISSUE #5605: remove everything @deprecated for MN-1 release
## What is the Problem Being Solved?

MN-1 launch requires some breaking changes. Let's bundle them together so we have fewer breaks after.

## Description of the Design

Including but not limited to,
- legacyMap.js

**Acceptance criteria**
- No `@deprecated` in repo
- Loadgen not broken (e.g. [using fallback](https://github.com/Agoric/testnet-load-generator/pull/76))

### PR DESCRIPTION
**Pull request by turadg**
closes: #5605

## Description

Removes all `@deprecated` from Inter Protocol. Two deprecations were reversed as they aren't worthwhile at this time,
[chore(reserve): un-deprecate getAllocations](https://github.com/Agoric/agoric-sdk/pull/7573/commits/877757330b05bee188ec400501f3c10f4dc15f07)
[chore(vaults): un-deprecate getSubscription](https://github.com/Agoric/agoric-sdk/pull/7573/commits/994752c9ecb8f0025121ac7d6d30b5ecf9e7c3bd) (it's part of the governance API)

### Security Considerations

Smaller API surface :) 

### Documentation Considerations

The "breaking changes" are to contracts that have never run in production.

### Testing Considerations

CI

### CODE REVIEW

File: docs/threat_models/vaultFactory/vaultFactory.puml [hunk 1]
```diff
@@ -17,9 +17,10 @@ node "CloseVaultInvitation"
 node "Vat" {
     node VaultFactory {
         node "publicAPI" {
-            circle getCollaterals
-            circle makeVaultInvitation
-            circle getRunIssuer
+            circle getCollateralManager
+            circle getMetrics
+            circle getDebtIssuer
+            circle getPublicTopics
```
Comment (Chris-Hibbert) [inline docs/threat_models/vaultFactory/vaultFactory.puml]: Why just this list?  I see quite a few
```
        getCollateralManager
        getCollaterals
        getMetrics
        getRunIssuer
        getSubscription
        getPublicTopics
        getElectorateSubscription
        getGovernedParams
        getInvitationAmount
```

**Review by Chris-Hibbert — APPROVED** [review]
I'm fine with this list. There are other @deprecated markers that apply to changes that will need a migration process.

File: docs/threat_models/vaultFactory/vaultFactory.puml (hunk 1 above)
Comment (turadg) [reply docs/threat_models/vaultFactory/vaultFactory.puml]: thanks, updated

**Review by Chris-Hibbert — APPROVED** [review]
LGTM

**Pull request merged**